samples/nrf9160/serial_lte_modem/src/slm_at_host.h

```diff
@@ -21,12 +21,21 @@
 /**@brief AT command handler type. */
 typedef int (*slm_at_handler_t) (enum at_cmd_type);
 
+/**@brief AT command list item type. */
 typedef struct slm_at_cmd_list {
 	u8_t type;
 	char *string;
 	slm_at_handler_t handler;
 } slm_at_cmd_list_t;
 
+/**@brief Shutdown modes. */
+enum shutdown_modes {
+	SHUTDOWN_MODE_NONE,
```

Comment:
Is there a use-case for `SHUTDOWN_MODE_NONE`? I don't see it and I find this entry quite confusing. If we need some value to indicate incorrect mode though I'd call it `SHUTDOWN_MODE_INVALID`, to clearly indicate it's not valid mode.

Reply:
Updated, thanks